osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/extraction_order_status.html

```diff
@@ -16,27 +16,31 @@
         </h2>
         <table>
             <body>
+                <tr><th>{% trans 'ID' %}</th><td>{{ extraction_order.id }}</td></tr>
+                <tr><th>{% trans 'Excerpt' %}</th><td>{{ extraction_order.excerpt.name }}</td></tr>
                 <tr>
-                    <tr><th>{% trans 'ID' %}</th><td>{{ extraction_order.id }}</td>
-                    <tr><th>{% trans 'Excerpt' %}</th><td>{{ extraction_order.excerpt.name }}</td>
-                    <tr>
-                        <th>{% trans 'Status' %}</th>
-                        <td>
-                            {% include 'excerptexport/partials/status.html' with status=extraction_order.get_state_display %}
-                            {% with extraction_order.get_state_display|lower as state %}
-                                {{ state }}
-                            {% endwith %}
-                        </td>
-                    </tr>
-                    {% if extraction_order.process_start_date %}
-                    <tr><th>{% trans 'Extraction started at' %}</th><td>{{ extraction_order.process_start_date|date:"r" }}</td>
-                    {% endif %}
-                    <tr><th>{% trans 'Issuer' %}</th><td>{{ extraction_order.orderer.get_username }}</td></tr>
+                    <th>{% trans 'Status' %}</th>
+                    <td>
+                        {% include 'excerptexport/partials/status.html' with status=extraction_order.get_state_display %}
+                        {% with extraction_order.get_state_display|lower as state %}
+                            {{ state }}
+                        {% endwith %}
+                    </td>
+                </tr>
+                {% if extraction_order.process_start_date %}
+                <tr><th>{% trans 'Extraction started at' %}</th><td>{{ extraction_order.process_start_date|date:"r" }}</td></tr>
+                {% endif %}
+                <tr><th>{% trans 'Issuer' %}</th><td>{{ extraction_order.orderer.get_username }}</td></tr>
             </body>
         </table>
 
-        {% if extraction_order.are_downloads_ready %}
-            <h3>{% trans 'Downloads' %} (Σ:{{ extraction_order.output_files.count }})</h3>
+        {% if extraction_order.output_files.count > 0 %}
+            <h3>{% trans 'Downloads' %}&nbsp;<!--
```

Comment:
Unfortunately it's not as cool as it sounds :-( . `spaceless` removes spaces between tags but not between tags and text. So it's not really what I'm looking for.
@njordan-hsr Is there something more agressive than `spaceless`?